code/modules/reagents/Chemistry-Reagents/Toxin-Reagents.dm

```diff
@@ -218,7 +226,7 @@
 	if(iscarbon(M))
 		var/mob/living/carbon/C = M
 		if(!C.wear_mask) // If not wearing a mask
-			var/damage = min(round(0.4*volume, 0.1),10)
+			var/damage = rand(volume,17)
```

Comment:
i don't think i meant this to be committed, but at one point i was trying to make the weed killer spray non-redundant by making it a decent weapon so it could be returned to the botany fertiliser vendor as a contraband item

currently we have weed killer spray that looks very similar to pest spray which isn't in the game, while weed killer's job is taken by the hoe thing which immediately removes all weeds from a tray in a single click